src/test/scala/mesosphere/marathon/api/v2/AppTasksResourceTest.scala

```diff
@@ -47,15 +47,16 @@ class AppTasksResourceTest extends MarathonSpec with Matchers with GivenWhenThen
     config.zkTimeoutDuration returns 5.seconds
     taskTracker.appTasksSync(appId) returns Set(task1, task2)
     taskKiller.kill(any, any) returns Future.successful(toKill)
+    groupManager.app(appId) returns Future.successful(Some(AppDefinition(appId)))
 
     val response = appsTaskResource.deleteOne(
-      appId.root, task1.taskId.idString, scale = false, force = false, auth.request, auth.response
```

Comment:
Only if the path would be equal to the root path (1 element) which is not true here.